src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -96,12 +99,24 @@ const RegistrationForm: FunctionComponent<Props> = ({
                     children: backendSupportChildren,
                     guardian: backendSupportGuardian,
                   },
-                });
-                history.push('/registration/success');
+                })
+                  .then(result => {
+                    history.push('/registration/success');
+                    console.log(result);
+                  })
+                  .catch(err => {
+                    toast('Failed to submit to server');
+                    if (err instanceof ApolloError) {
```

Comment:
is this follow any practice at all in Kuva ? just toast it i think, this check is not need if `catch` got it

Reply:
I think some errors, like the user not having a network connection, should not go to Sentry, maybe? It's not like they are bugs we can fix.